bw2io/export/gexf.py

```diff
@@ -10,16 +10,52 @@
 
 
 class DatabaseToGEXF(object):
-    """Export a Gephi graph for a database.
+    # """Export a Gephi graph for a database.
```

Comment:
...same question here: [why are you keeping the old docstrings in and commenting them out?](https://github.com/brightway-lca/brightway2-io/pull/169/files#r1141269894)